runner/debug/simple.go

```diff
@@ -0,0 +1,101 @@
+package debug
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/hcdiag/op"
+	"github.com/hashicorp/hcdiag/redact"
+	"github.com/hashicorp/hcdiag/runner"
+)
+
+var _ runner.Runner = SimpleDebug{}
+
+// The "Simple" debug wrapper can be used for Nomad, Vault, and Consul.
+// It only deals with flags common to all three 'debug' commands
+type SimpleDebug struct {
```

Comment:
Do we use or intend to use SimpleDebug for Nomad and Consul, or is the intent to give them their own debug runners like Vault? If it's the former than we should call it `Debug` or `ProductDebug` and if it's the latter then we shouldn't merge it in.

Reply:
Good point -- I've created tasks/stories for NomadDebug and VaultDebug, and was on the fence about keeping SimpleDebug. I can't think of a great reason to keep it, either, if you think the current (VaultDebug) approach works well.